config/routes.rb

```diff
@@ -137,8 +137,10 @@
       end
 
       scope '/subject-knowledge' do
-        get '/' => 'subject_knowledge#edit', as: :edit_subject_knowledge
-        patch '/' => 'subject_knowledge#update'
+        get '/' => 'subject_knowledge#new', as: :new_subject_knowledge
+        patch '/' => 'subject_knowledge#create'
```

Comment:
`post`?

Reply:
It's just updating an attr on the application form though right?